src/Components/Reports/Common/ExecutiveSecondPage.js

```diff
@@ -0,0 +1,140 @@
+import React, { Fragment } from 'react';
+import {  Dl, Dt, Dd } from '@redhat-cloud-services/frontend-components-pdf-generator';
+import { Canvas, Text, View } from '@react-pdf/renderer';
+import { TachometerAltIconConfig } from '@patternfly/react-icons/dist/js/icons/tachometer-alt-icon';
+import { AngleDoubleDownIconConfig } from '@patternfly/react-icons/dist/js/icons/angle-double-down-icon';
+import { AngleDoubleUpIconConfig } from '@patternfly/react-icons/dist/js/icons/angle-double-up-icon';
+import { AutomationIconConfig } from '@patternfly/react-icons/dist/js/icons/automation-icon';
+import { InProgressIconConfig } from '@patternfly/react-icons/dist/js/icons/in-progress-icon';
+import styles from './styles';
+
+export const ExecutiveSecondPage = () => {
+
+    return <Fragment key="second-page">
+        <Text style={styles.execHeading}>Description of states</Text>
+        <Dl>
+            <Dt>
+                <View style={{ display: 'flex',
+                    flexDirection: 'row' }}>
+                    <Canvas
```

Comment:
`CanvasComponent` can be used here as well. (we can use different name though 😅  )